src/cgutils.cpp

```diff
@@ -4280,8 +4280,9 @@ static jl_cgval_t emit_memoryref(jl_codectx_t &ctx, const jl_cgval_t &ref, jl_cg
             ovflw = ctx.builder.CreateICmpUGE(ctx.builder.CreateAdd(offset, mlen), ctx.builder.CreateNUWAdd(mlen, mlen));
         }
 #endif
+        //Is this change fine
```

Comment:
It looks like we must be able to do this arithmetic later, and while we know this computation of `bidx0` cannot overflow (because of checking `ovflw`), we don't appear to know that the intermediate step won't overflow:
```
            Value *bidx0 = ctx.builder.CreateSub(
                ctx.builder.CreatePtrToInt(newdata, ctx.types().T_size),
                ctx.builder.CreatePtrToInt(mptr, ctx.types().T_size));
```